### DISCUSSION TITLE
Нет баллов

### DISCUSSION THREAD

**Discussion by luck-o-o-luck** [is-oop-y25/TemplateRepository#3] [shreks-issues] [answered]
Ну вот просто НЕТ баллов в таблице. Хотя в пр пишет, что 4 из 4: https://github.com/is-oop-y25/Bansai3/pull/3

**Comment by FrediKats** [comment, marked as answer]
Были проблемы с фоновым обновлением таблицы. Данные про выставленные баллы были сохранены в базу. Сейчас проблему устранили, таблица перегенерировалась и баллы появились.
